wizeline/remix-project-lab
# Create project functional inputs

# What does this PR do?

Migrate the simple inputs to create a project



# Where should the reviewer start?

You can start checking ProjectForm.tsx and review every input in there 


# How should this be manually tested?

1. From the landing page click the button of 'New Proposal'
2. Fill the inputs 
3. Click the 'Create Project' button 
4. Check in your terminal your inputs from the console.log


# Any background context you want to provide?

There are still some inputs that are needed to create a project that are commentated.
This PR doesn't include the useFetcher for creating the project, just the base inputs.



# What are the relevant tickets?

https://github.com/wizeline/remix-project-lab/pull/48


# Screenshots
![image](https://user-images.githubusercontent.com/42359482/195907605-e09e24cd-d3e1-4855-88c0-a9e975adac6d.png)
![image](https://user-images.githubusercontent.com/42359482/195910687-70789ff5-f47f-4413-ba8e-5a453ed4f2d5.png)
![image](https://user-images.githubusercontent.com/42359482/195910817-377d3f2f-1127-4891-8b56-7446f0b89c94.png)



This PR was made with the collaboration of: @dmojarrot 
# Checklist
